yarn-project/archiver/src/archiver/archiver.test.ts

```diff
@@ -126,15 +136,35 @@ describe('Archiver', () => {
       });
     });
 
-    rollupRead = mock<MockRollupContractRead>();
-    rollupRead.archiveAt.mockImplementation((args: readonly [bigint]) =>
+    mockRollupRead = mock<MockRollupContractRead>();
+    mockRollupRead.archiveAt.mockImplementation((args: readonly [bigint]) =>
       Promise.resolve(blocks[Number(args[0] - 1n)].archive.root.toString()),
     );
+    mockRollupEvents = mock<MockRollupContractEvents>();
+    mockRollupEvents.L2BlockProposed.mockImplementation(async (filter: any, { fromBlock, toBlock }) => {
+      return await Promise.resolve(
+        l2BlockProposedLogs.filter(log => log.blockNumber! >= fromBlock && log.blockNumber! <= toBlock),
+      );
+    });
```

Comment:
```suggestion
    mockRollupEvents.L2BlockProposed.mockImplementation((filter: any, { fromBlock, toBlock }) => 
      Promise.resolve(
        l2BlockProposedLogs.filter(log => log.blockNumber! >= fromBlock && log.blockNumber! <= toBlock),
      ));
```

Just for readability (and OCD)